templates/github/.github/workflows/scripts/before_script.sh.j2

```diff
@@ -29,6 +29,14 @@ if [[ "$TEST" == 'pulp' || "$TEST" == 'performance' || "$TEST" == 'upgrade' || "
   cmd_prefix dnf install -yq lsof which dnf-plugins-core
 fi
 
+{%- if disabled_redis_runners %}
+read -ra REDIS_DISABLED <<< "$REDIS_DISABLED"
+if [[ " ${REDIS_DISABLED[*]} " =~ " ${TEST} " ]]; then
+  cmd_prefix bash -c "s6-svc -d /var/run/s6/services/redis"
+  echo "The Redis service was disabled for $TEST"
+fi
+{%- endif %}
```

Comment:
Do we have a way to disable Redis also? Or is Pulp just able to fallback if missing?

Reply:
This disables the Redis server service. What do you mean by `disable Redis also`? The idea, that comes from https://pulp.plan.io/issues/8997, was to simulate the Pulp's behaviour after disabling the Redis service. The actual change to the template allows plugin writers to test their plugins in such a state as well.

Reply:
I'd say this shuts the service down, simulating a failure. But what if a user wants to run Pulp without ever installing / configuring redis. Would this use different codepaths in any way? Do we not support such a thing yet?

Reply:
There is a separate issue filed for that: https://pulp.plan.io/issues/9475. Should I rather focus on that one in this PR instead?

Reply:
No, just wanted to know if we miss an opportunity today.